static/inline/js/inline.js

```diff
@@ -219,7 +220,8 @@
                         breakpoint: 769,
                         settings: {
                             arrows: false,
-                            centerPadding: '0px',
+                            centerPadding: Utils.isset(generalOptions.product_holder_slide_center_padding) ? generalOptions.product_holder_slide_center_padding :'25% 0 0',
```

Comment:
Shouldn't the option `product_holder_slide_center_padding` default value actually be "25% 0 0"?
This way you just pass the option and that's it.

`centerPadding: generalOptions.product_holder_slide_center_padding`

Reply:
👍  will update